src/main/kotlin/org/taktik/icure/be/ehealth/logic/kmehr/smf/impl/v23g/SoftwareMedicalFileImport.kt

```diff
@@ -932,6 +932,7 @@ class SoftwareMedicalFileImport(val patientLogic: PatientLogic,
 
     private fun extractTags(item: ItemType): Collection<CodeStub> {
         return (item.cds.filter { it.s == CDITEMschemes.CD_PARAMETER || it.s == CDITEMschemes.CD_LAB || it.s == CDITEMschemes.CD_TECHNICAL }.map { CodeStub(it.s.value(), it.value, it.sv) } +
+                item.cds.filter { (it.s == CDITEMschemes.LOCAL && it.sl.equals("LOCAL-PARAMETER")) }.map { CodeStub(it.sl, it.value, it.sv) } +
```

Comment:
`LOCAL-PARAMETER` is hardcoded

Reply:
This is a question for AD, I cannot extract it without introducing design changes that might not align with his guidelines.